source/_integrations/snapcast.markdown

```diff
@@ -74,5 +74,5 @@ Set the latency of a speaker.
 
 | Service data attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
-| `entity_id` | yes | String or list of `entity_id`s for which latency will be adjusted.
-| `latency` | yes | Latency in ms.
+| `entity_id` | no | String or list of `entity_id`s for which latency will be adjusted.
```

Comment:
`entity_id` is optional, so please revert the change on this line.

Reply:
Either entity_id or area_id is required.

Reply:
I'm a bit confused on what to do now. Should I add add 'area_id' an make both 'entity_id' and 'area_id' optional? Just want to make sure it's clear. Thanks

Reply:
We currently do not document the use of area_id.
A) General documentation around it is missing
B) It is not really useable manually (finding the area ID to use is hard).

I've adjusted the PR to correct the entity_id.